testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -60,16 +61,26 @@ class TopologyAttrMixin(object):
     Ridx = np.array([0, 0, 2, 2, 1, 1, 3, 3, 1, 2])
     Sidx = np.array([0, 1, 1, 0])
 
+    # single_idx = np.array([0])
+
     def setUp(self):
         self.top = Topology(10, 4, 2,
                             attrs=[self.attrclass(self.values.copy())],
                             atom_resindex=self.Ridx,
                             residue_segindex=self.Sidx)
-
         self.attr = getattr(self.top, self.attrclass.attrname)
 
+        # TODO: should we treat single element attr specially. We do that
```

Comment:
Yes my question is how to test for this bug so that we are sure all behave the same. My first guess was to only test for a single element when `type(rg._ix) == int`, that path isn't covered yet.